### PR TITLE
[Velero] Enable prometheus metrics by default

### PR DESCRIPTION
**Pull request by ashish-amarnath**
Signed-off-by: Ashish Amarnath <ashisham@vmware.com>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/velero/values.yaml [hunk 1]
```diff
@@ -42,7 +42,7 @@ extraVolumeMounts: []
 
 # Settings for Velero's prometheus metrics. Disabled by default.
```
Comment (skriss) [inline stable/velero/values.yaml]:
```suggestion
 # Settings for Velero's prometheus metrics. Enabled by default.
```

Comment (ashish-amarnath) [reply stable/velero/values.yaml]: Thanks for catching that!

**Review by skriss — APPROVED** [review]
LGTM!

**Pull request merged**